lib/pquot.gi

```diff
@@ -1310,7 +1310,9 @@ end );
 
 #############################################################################
 ##
-#F  AbelianPQuotient  . . . . . . . . . . .  initialize an abelian p-quotient
+#F  AbelianPQuotient  . . . . . . try to initialize an abelian p-quotient
+##                    . . . . . . return true if we are sucessful
+##                    . . . . . . return false otherwise
```

Comment:
That's not how we format function comments.

There is (at most) one line with `. . . ` that gives very brief summary, which is "right aligned"

Then optionally one can have text below that gives details.
```suggestion
#F  AbelianPQuotient  . . . . . . . . try to initialize an abelian p-quotient
##
##  Return true if we are successful, return false otherwise.
```

Reply:
Ah, I didn't know. Thanks for explaining